README.md

```diff
@@ -76,6 +76,17 @@ MyType.prototype[fl.map] = MyType.prototype.map
 var foo = bar[fl.map](x => x + 1)
 ```
 
+## Type representatives
+
+Certain behaviours are defined from the perspective of a member of a type.
+Other behaviours do not require a member. Thus certain algebras require a
+type to provide a value-level representative (with certain properties). The
+Identity type, for example, could provide `Id` as its type representative:
+`Id :: TypeRep Identity`.
```

Comment:
> The part which confused me was `Id :: TypeRep Identity` (`Id` and `Identity` are different when `Number` and `Number` are same) is it because we assume definition of Identity would be something like this `type Identity a = Id a`?

When we talk about "the Identity type" we're referring to a concept with no manifestation in JavaScript. When we talk about "`Id`" we're referring to a real thing: an identifier we could evaluate in a REPL. `Id` lives in the world of values to the left of the colons; `Identity` lives in the world of types to the right of the colons.

Since the world of values and the world of types are separate, we needn't worry about naming collisions. This is valid:

```haskell
Identity :: a -> Identity a
```

`Identity` on the left of the `::` is a value, whereas `Identity` on the right is a type (a type constructor, actually).

Similarly, we could name our type representative `Identity` rather than `Id` without conflict:

```haskell
Identity :: TypeRep Identity
```

I hope this clarifies things a little. :)